worlds/dark_souls_3/docs/en_Dark Souls III.md

```diff
@@ -7,19 +7,22 @@ config file.
 
 ## What does randomization do to this game?
 
-In Dark Souls III, all unique items you can earn from a static corpse, chest, or the death of a Boss/NPC are
-randomized.
-An option is available from the settings page to also randomize upgrade materials, Estus shards, and consumables.
-Another option is available to randomize the level of the generated weapons (from +0 to +10/+5).
+Items that can be picked up from static corpses, taken from chests, or earned from defeating enemies or NPCs can be
+randomized. Common pickups like titanite shards or firebombs can be randomized as "progressive" items. That is, the
+location "Titanite Shard #5" is the fifth titanite shard you pick up, no matter where it was from. This is also what
+happens when you randomize Estus Shards and Undead Bone Shards.
 
-To beat the game, you need to collect the 4 "Cinders of a Lord" randomized in the multiworld
-and kill the final boss "Soul of Cinder."
+It's also possible to randomize the upgrade level of weapons and shields as well as their infusions (if they can have
+one), and there are settings that can make the randomized experience more convenient or more interesting, such as
```

Comment:
```suggestion
It's also possible to randomize the upgrade level of weapons and shields as well as their infusions (if they can have
one.) Additionally, there are settings that can make the randomized experience more convenient or more interesting, such as
```

Reply:
Just a small suggestion for sentence flow. Besides this, looks good!

Reply:
Does the period belong inside the parentheses? It looks odd to me, but I know that's the rule for quotation marks.

Reply:
Mixed up rules for parentheses, suggestion is now correct

Reply:
Ah, pointed out in Discord. Sounds good to me.